src/main/java/seedu/address/model/widget/WidgetObject.java

```diff
@@ -45,11 +46,27 @@ public WidgetObject() {
      * @param args All the fields to be set.
      */
     public void set(String ...args) {
+        // Sets all fields from start to end order.
         for (int i = 0; i < Math.min(args.length, 14); i++) {
             divs[i] = args[i];
         }
     }
```

Comment:
This method needs more explanation using comments/better descriptive names. For example, what is start to end order? Why is there the Math.min with the magic literal 14? 

Would suggest extracting out the 14 as a private final static var

Also, I think it should be `String... args` and not `String ...args`? At least that's the format google java format uses